scene/2d/area_2d.cpp

```diff
@@ -143,8 +143,14 @@ void Area2D::_body_enter_tree(ObjectID p_id) {
 
 	E->value.in_tree = true;
 	emit_signal(SceneStringNames::get_singleton()->body_entered, node);
+	if (node && node->has_signal(SceneStringNames::get_singleton()->entered_area)) {
```

Comment:
```suggestion
	if (node->has_signal(SceneStringNames::get_singleton()->entered_area)) {
```
In the cases where no `if (node)` checks were added before this is unnecessary

Reply:
Ah, alright, will do.

Reply:
Done!
idk how I got 
```C++
if (node) {
	if (node && [...]
```
Must be losing focus, sorry ^^'